ddlitlab2024/ml/training/train.py

```diff
@@ -137,8 +136,8 @@
             lr_scheduler.step()
             ema.update()
 
-            if (batch + 1) % 100 == 0:
-                print(f"Epoch {epoch}, Loss: {mean_loss / batch}, LR: {lr_scheduler.get_last_lr()[0]}")
+            if (i + 1) % 5 == 0:
+                print(f"Epoch {epoch}, Loss: {mean_loss / i}, LR: {lr_scheduler.get_last_lr()[0]}")
```

Comment:
How about using the logger or is there a reason no to?

Reply:
The code predates the logger, so we can integrate it in the future. Together with other things like wandb.